packages/radio-group/src/RadioGroup.tsx

```diff
@@ -163,6 +163,8 @@ export default class RadioGroup extends PureComponent<
           `,
           className,
         )}
+        role="group"
+        aria-label={this.defaultName}
```

Comment:
Shouldn't this be `name`, not `this.defaultName` here? We set the default for `name` to `this.defaultName`.